apps/etterlatte-saksbehandling-ui/client/src/components/behandling/vilkaarsvurdering/Vilkaarsvurdering.tsx

```diff
@@ -66,8 +85,25 @@ export const Vilkaarsvurdering = (props: { behandling: IBehandlingReducer }) =>
         </HeadingWrapper>
       </ContentHeader>
 
-      {behandlingId && vilkaarsvurdering && (
+      {behandlingId && vilkaarsvurdering && !isPending(slettVilkaarsvurderingStatus) && (
         <>
+          {behandles && visHarGammelVilkaarsvurdering() && (
```

Comment:
Siden `behandles` alltid skal sjekkes ifm. `visHarGammelVilkaarsvurdering()` kan den kanskje for ryddighetens skyld flyttes til funksjonen?

Reply:
Enig 👍